src/shared/components/TruncatedTextSVG.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from "react";
```

Comment:
should we add tooltip to the name of this component?  also, would it make sense to somehow combine this with victory label so we could use it elsewhere?

Reply:
yeah, this is victory compatible. it's a plug in as `labelComponent`